recipes/opene57/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+    default_options = {
+        'with_tools': False,
+        'shared': False,
+        'fPIC': True}
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "7",
+            "clang": "6",
+            "apple-clang": "10",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        if self.options.shared:
+            raise ConanInvalidConfiguration("OpenE57 cannot be built as shared library yet")
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 17)
+        minimum_version = self._minimum_compilers_version.get(str(self.settings.compiler), False)
+        if not minimum_version:
+            self.output.warn("C++17 support required. Your compiler is unknown. Assuming it supports C++17.")
+        elif tools.Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration("C++17 support required, which your compiler does not support.")
+
+    def build_requirements(self):
+        if self.options.with_tools:
+            self.build_requires("boost/1.78.0")
+            self.options["boost"].multithreading = True
+            self.options["boost"].shared = False
+
+    def requirements(self):
+        if self.settings.os == "Linux" or tools.is_apple_os(self.settings.os):
+            self.requires("icu/70.1")
+            self.options["icu"].shared = False
+
+        self.requires("xerces-c/3.2.3")
+        self.options["xerces-c"].shared = False
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_EXAMPLES"] = False
+        self._cmake.definitions["BUILD_TOOLS"] = self.options.with_tools
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_WITH_MT"] = "MT" in str(msvc_runtime_flag(self))
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="LICENSE.libE57", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        os.remove(os.path.join(self.package_folder, "CHANGELOG.md"))
```

Comment:
```suggestion
        os.remove(os.path.join(self.package_folder, "CHANGELOG.md"))
        tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), "*.dll")
```

Need to delete runtime libraries. They should be consumed from system instead.